pkgs/development/python-modules/blobfile/default.nix

```diff
@@ -0,0 +1,45 @@
+{ buildPythonPackage
+, fetchPypi
+, setuptools
+, urllib3
+, pycryptodomex
+, lxml
+, filelock
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "blobfile";
+  version = "2.0.1";
+
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    dist = "py3";
+
+    hash = "sha256-b1Gz6UjzCpLnNKl0sk/ND2pRhB/Qg96WiJkjFIE1jaI=";
+  };
+
+  propagatedBuildInputs = [
+    urllib3
+    pycryptodomex
+    lxml
+    filelock
+  ];
+
+
+  doCheck = false;
```

Comment:
Please add a comment explaining why checks have to be disabled

I took a quick look and it seems like the pytest stuff in the package expects to have internet access and the stuff in the `testing` directory aren't otherwise that useful since they just call (optionally) docker, a (optional) formatter, a type checker, and then pytest. If you had other reasons though please add them